src/jaxsim/mujoco/__main__.py

```diff
@@ -0,0 +1,191 @@
+    )
+
+    parser.add_argument(
+        "-v",
+        "--visualize",
+        action="store_true",
+        default=False,
+        help="Visualize the description in the Mujoco viewer (default: %(default)s).",
+    )
+
+    parser.add_argument(
+        "-b",
+        "--base-position",
+        metavar=("x", "y", "z"),
+        nargs=3,
+        type=float,
+        default=None,
+        help="Override the base position (supports only floating-base models).",
+    )
+
+    parser.add_argument(
+        "-q",
+        "--base-quaternion",
+        metavar=("w", "x", "y", "z"),
+        nargs=4,
+        type=float,
+        default=None,
+        help="Override the base quaternion (supports only floating-base models).",
+    )
+
+    args = parser.parse_args()
+
+    # ==================
+    # Validate arguments
+    # ==================
+
+    # Expand the path of the URDF/SDF file if not absolute.
+    if args.description is not None:
+        args.description = (
+            (
+                args.description
+                if args.description.is_absolute()
+                else pathlib.Path.cwd() / args.description
+            )
+            .expanduser()
+            .absolute()
+        )
+
+        if not pathlib.Path(args.description).is_file():
+            msg = f"The URDF/SDF file '{args.description}' does not exist."
+            parser.error(msg)
+            exit(1)
+
+    # Expand the path of the output MJCF file if not absolute.
+    if args.export is not None:
+        args.export = (
+            (
+                args.export
+                if args.export.is_absolute()
+                else pathlib.Path.cwd() / args.export
+            )
+            .expanduser()
+            .absolute()
+        )
+
+        if pathlib.Path(args.export).is_file() and not args.force:
+            msg = "The output file '{}' already exists, use '--force' to override."
+            parser.error(msg.format(args.export))
+            exit(1)
+
+    # ================================================
+    # Load the URDF/SDF file and produce a MJCF string
+    # ================================================
+
+    match args.description.suffix.lower()[1:]:
+
+        case "urdf":
+            mjcf_string, assets = UrdfToMjcf().convert(urdf=args.description)
+
+        case "sdf":
+            mjcf_string, assets = SdfToMjcf().convert(
+                sdf=args.description, model_name=args.model_name
+            )
+
+        case _:
+            msg = f"The file extension '{args.description.suffix}' is not supported."
+            parser.error(msg)
+            exit(1)
+
+    if args.print:
+        print(mjcf_string, flush=True)
+
+    # ========================================
+    # Write the MJCF string to the output file
+    # ========================================
+
+    if args.export is not None:
+        with open(args.export, "w+") as file:
```

Comment:
```suggestion
        with open(args.export, "w+", encoding="utf-8") as file:
```
This could create issues when using Windows, see https://peps.python.org/pep-0597/

Reply:
Good catch, addressed in https://github.com/ami-iit/jaxsim/pull/83/commits/4dcd4a760ac2d9afe7d6a74fea385f2447ebb07f